tutor/specs/screens/new-course/select-dates.spec.jsx

```diff
@@ -8,6 +8,9 @@ jest.mock('../../../src/models/user', () => ({
 
 describe('CreateCourse: Selecting course dates', function() {
   let ux;
+
+  TimeMock.setTo('2018-12-01T12:00:00.000Z');
+
```

Comment:
this is unrelated to the PR, but the spec failed once the year rolled over to 2019

Reply:
nah, it's important to get into master ASAP, so that specs don't continue to fail

Reply:
Ahhhh I thought you were saying that it was failing with the fix. I get it now. Thanks!